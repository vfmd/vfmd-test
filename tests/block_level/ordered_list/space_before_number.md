Normal text

1. No space

Normal text

 1. One space

Normal text

  1. Two space

Normal text

   1. Three space

Normal text

    1. Four space

