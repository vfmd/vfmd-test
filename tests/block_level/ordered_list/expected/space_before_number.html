<p>Normal text</p>

<ol>
<li>No space</li>
</ol>

<p>Normal text</p>

<ol>
<li>One space</li>
</ol>

<p>Normal text</p>

<ol>
<li>Two space</li>
</ol>

<p>Normal text</p>

<ol>
<li>Three space</li>
</ol>

<p>Normal text</p>

<pre><code>1. Four space
</code></pre>
